src/core/sys/linux/perf_event.d

```diff
@@ -0,0 +1,2468 @@
+/**
+  * D header file for perf_event_open system call.
+  *
+  * Converted from linux userspace header, comments included.
+  *
+  * Authors: Max Haughton
+  */
+module core.sys.linux.perf_event;
+version (linux)  : extern (C):
+@nogc:
+nothrow:
+@system:
+
+import core.sys.posix.sys.ioctl;
+import core.sys.posix.unistd;
+
+version (X86_64)
+{
+        enum __NR_perf_event_open = 298;
+}
+else version (X86)
+{
+        enum __NR_perf_event_open = 336;
+}
+else version (ARM)
+{
+        enum __NR_perf_event_open = 364;
+}
+else version (AArch64)
+{
+        enum __NR_perf_event_open = 241;
+}
+else
+{
+        static assert(0, "Architecture not supported");
+}
```

Comment:
We should really be rejecting PRs that don't cover all primary supported linux platforms.  It's annoying to be running through some finishing line bootstrap tests, and I can't even build druntime anymore on PPC.

Reply:
Well that's why we need to extend the CI here to include you.